### DISCUSSION TITLE
Request a Phanta DBs without viral genomes

### DISCUSSION THREAD

**Discussion by YiyanYang0728** [bhattlab/phanta#45] [Phanta DBs] [answered]
Dear Phanta Team,

I hope this message finds you well. First and foremost, thank you for providing such a wonderful tool. It has been incredibly useful for my research.

I am reaching out with a specific request: I am interested in obtaining the kraken databases like unmasked_db_v1.01.tar.gz and masked_db_v1.tar.gz but without the inclusion of viruses. If such databases have already been established, I would greatly appreciate your assistance in providing access to them. My goal is to compare these databases with the standard Phanta databases to examine differences in microbial profiles.

Thank you in advance for your time and help with this request.

Best regards,
Yiyan

**Comment by meenachakra** [comment]
Hi Yiyan, so glad to hear that Phanta has been useful to your research! Just to clarify your question, would you like to keep the exact same microbes in the database, simply excluding the viruses? Or do you just wish to have any Kraken2 database that focuses on bacteria?

**Reply by YiyanYang0728** [reply]
Hi Meena,
Thank you for your prompt response. A database that retains the exact same microbes (including Archaea, Bacteria, and others) but excludes viruses would be ideal.
Thanks for your assistance again.

**Comment by yipinto** [comment, marked as answer]
HI Yiyan,
Thanks for using Phanta.
We don't have such a database, so you'll need to create it as a custom db, see instructions [here](https://github.com/bhattlab/phanta/blob/main/databases.md).
The good news are that you have the taxonomy files ready from the default DB, which is usually the most difficult part, so you are only few commands away from making it.
Please notice that since Phanta major aim was to detect viruses alongside to bacteria, we never tried to run Phanta in the absence of viral sequences, so Phanta run might not finish successfully and it will for sure affect the post-processing scripts.
If you need any further assistance with sharing fna files, discussing how is the best way to get your db, don't hesitate to email us. https://www.bhattlab.com/people.

**Reply by YiyanYang0728** [reply]
Hi @yipinto ! Thank you so much for your instructions and your willingness to offer assistance! I followed the custom database  tutorial and successfully built the DB myself. The most challenging part was indeed preparing the taxonomy files, which took quite a bit of time. I just wish I had known about it sooner (crying_face.jpg). Thanks again for the message.

**Reply by yipinto** [reply]
Happy that you successfully created the DB! and for future users- you could use the taxonomy files from the default db!
That's seem like a very interesting usage of Phanta, also interested in the masking decisions you have made, if you want to discuss it in the future don't hesitate to email us.
Yishay

**Reply by YiyanYang0728** [reply]
Yes, I’ve used and modified the nodes.dmp and names.dmp files from HumGut, which are of great help, but I couldn’t find these files in the downloaded Phanta databases. As for the masking, I followed Kraken’s default masking steps, but no prophage masking was applied. It would be fantastic if the prophage-masked HumGut sequences (.fna files) could be made available. I would greatly appreciate your help with this.

Best,
Yiyan

**Reply by meenachakra** [reply]
Hi Yiyan! what is your timeline? If you are aiming to do this quickly, would it be possible for you to follow the methods listed in our paper, under "Masking prophages in prokaryotic genomes"? We can certainly make the .fna files available, however this requires a bit of work on our part and we are both a bit busy for the next few days so wanted to point you towards the methods as well, in case faster :) 

https://www.nature.com/articles/s41587-023-01799-4#Sec19

**Reply by YiyanYang0728** [reply]
Thank you for getting back to me! There's no rush on this—I'm not working with a tight deadline, so feel free to take your time. I’ll take a look at the methods you suggested in the paper and see if I can proceed from there in the meantime.
I really appreciate your help and guidance!
